### ISSUE TITLE
Pipenv using wrong Python path

### ISSUE THREAD

**Issue by gardenerik** [sublimelsp/LSP-pyright#332]
Hello, when using LSP-pyright with Pipenv (2023.12.1) with a project that contains a `.env` file, it fails to correctly detect the Python executable, due to a unrelated log message from Pipenv:

```
LSP-pyright: INFO: Pipfile detected. Run subprocess command: pipenv --py
LSP-pyright: INFO: Using python path "Loading .env environment variables...
/home/gardener/.local/share/virtualenvs/bill-gl5kfYRG/bin/python"
```

Note the `Loading .env environment variables...` in the Python path. It seems like that is printed to stderr.

**Comment by jfcherng** [comment]
I can reproduce this when there is both `.env` and `Pipfile` exists at the same time.

- `Loading .env environment variables...` is printed to stderr.
- `/home/gardener/.local/share/virtualenvs/bill-gl5kfYRG/bin/python` is printed to stdout.

**Comment by jfcherng** [comment]
Should have been fixed in v1.3.7

**Comment by gardenerik** [comment]
Wow... That was quick. Thank you very much!
